rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -896,10 +1168,12 @@ extern "C"
 
         waitset->fixed_guard_conditions = fixed_guard_conditions;
         if (fixed_guard_conditions) {
+/*
```

Comment:
Any reason why this was commented? It seems like it should either be uncommented or deleted. It seems like important error checking to me...